mmedit/datasets/pipelines/augmentation.py

```diff
@@ -895,3 +895,42 @@ def __repr__(self):
         repr_str = self.__class__.__name__
         repr_str += f'(keys={self.keys}, reverse_ratio={self.reverse_ratio})'
         return repr_str
+
+
+@PIPELINES.register_module()
+class MirrorExtend:
```

Comment:
How about `MirrorSequenceExtend`, so that to rule out the wrong guess of spatial extention.

Reply:
This one seems more succinct. Will change~Thanks~